tutorials/vr/developing_for_oculus_quest.rst

```diff
@@ -0,0 +1,127 @@
+.. _doc_developing_for_oculus_quest:
+
+Developing for Oculus Quest
+===========================
+
+Introduction
+------------
+
+This tutorial goes over how to get started developing for the
+Oculus Quest with an official plugin.
+
+Before getting started there are two things you need to do.
+
+First you need to go through the steps on the :ref:`doc_exporting_for_android`
+page. This leads you through installing the toolset that Godot
+needs to export to Android devices.
+
+Next you need the Quest Plugin. You can download it from the Asset
+Library or manually download it from `here <https://github.com/GodotVR/godot-oculus-mobile-asset>`__.
```

Comment:
Looks like the link is incorrectly formatted?

Reply:
That seems correct to me. Double underscores can be used to create unnamed references, so you won't get a Sphinx warning if the same link is used more than once in a page.